docs/docs/references/configuration/cli/trivy_image.md

```diff
@@ -66,12 +77,22 @@ trivy image [flags] IMAGE_NAME
       --helm-values strings               specify paths to override the Helm values.yaml files
   -h, --help                              help for image
       --ignore-policy string              specify the Rego file path to evaluate each vulnerability
-      --ignore-status strings             comma-separated list of vulnerability status to ignore (unknown,not_affected,affected,fixed,under_investigation,will_not_fix,fix_deferred,end_of_life)
+      --ignore-status strings             comma-separated list of vulnerability status to ignore
+                                          Allowed values:
+                                            - unknown
+                                            - not_affected
+                                            - affected
+                                            - fixed
+                                            - under_investigation
+                                            - will_not_fix
+                                            - fix_deferred
+                                            - end_of_life
+                                          
```

Comment:
nit:
can we remove empty line (when there are no default values)?

Reply:
done https://github.com/aquasecurity/trivy/pull/8560/commits/d4d7a9cc216427e3a50d89bda0acc5fa0b2256c5